### PR TITLE
GFORMS-752 welsh translation

### PR DESCRIPTION
**Pull request by girishkamat**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -111,7 +111,9 @@ generic.shortText.error.pattern=Gall {0} ond cynnwys llythrennau, rhifau, bylcha
 generic.longText.error.pattern=Ni all {0} gynnwys y cymeriadau {1}
 generic.referenceNumber.error.pattern=Gall {0} ond cynnwys rhifau
 generic.charactersRemaining=Mae gennych {0} o gymeriadau yn weddill
+generic.characterRemaining=You have {0} character remaining
```
Comment (josef-vlach) [inline conf/messages.cy:114]: Cannot we copy&paste (or rather reverse engineered) welsh from this: https://github.com/hmrc/play-frontend-hmrc/blob/master/src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/hmrcCharacterCount.scala.html
?

Comment (girishkamat) [reply conf/messages.cy:114]: Thats a great idea, I was about to send it for translation. Now I dont have to

Comment (girishkamat) [reply conf/messages.cy:114]: The messages seem to be different though, nothing compared to what GDS provides

"You can enter up to...."

Comment (josef-vlach) [reply conf/messages.cy:114]: If are going to ultimately switch to `HmrcCharacterCount` then it should be ok. But I'll leave that to you to decide.

**Review by josef-vlach — APPROVED** [review]

**Pull request merged**